website_quote_substitute_product/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+{
+    'name': 'Substitute Product for Online Proposals',
+    'summary': 'Allows to change products by others recommended via web.',
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'Domatix, Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/sale-workflow',
+    'category': 'Sale',
+    'depends': ['website_quote'],
+    'data': [
```

Comment:
@carlosDomatix IMO  `data` sequence should be [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L29)